src/app.py

```diff
@@ -35,6 +38,31 @@
 
 display_header(st, m, p)
 
+#Upload file
+uploaded_file = st.file_uploader("Choose a XLSX file", type="xlsx")
+if uploaded_file:
+    result=[]
+    df=pd.read_excel(uploaded_file)
+    excel_filename=f"{p.current_date}_projected_census.xlsx"
+    xlsx_io = io.BytesIO()
+    writer = pd.ExcelWriter(xlsx_io, engine='xlsxwriter')
+    # book = xlwt.Workbook()
+    # excelWriter = pd.ExcelWriter(excel_filename)
+    for index, row in df.iterrows():
+        batch_param=getParamFromFile(row,d)
+        batch_model=SimSirModel(batch_param)
+        file_name_hosp=row["HOSPITAL_NAME"]
+        scenario=row["SCENARIO_LABEL"]
+        sheet_name=f"{file_name_hosp}_{scenario}_projected_admits"
```

Comment:
Change sheet name to 
 sheet_name=f"{file_name_hosp}_{scenario}_projected_census"